CONTRIBUTING.md

```diff
@@ -61,21 +60,15 @@ erroneously.
 The tests are executed automatically for each PR using the ci (see
 `.github/workflow/lint.yml` and `.github/workflow/tests.yml`).
 
-Running the tests manually:
+Running the tests,
 
-* Using patroni's nominal replica state of `streaming` (since v3.0.4):
+* manually:
 
   ```bash
-  pytest ./tests
+  pytest tests
```

Comment:
```suggestion
  pytest tests
  pytest --cov=check_patroni tests/    # includes code coverage info
```

Reply:
There is no need to specify a value for `--cov` as this is configured in `.coveragerc`; but I'll add the option still here.